### ISSUE TITLE
Cannot Install carbon-website with Yarn on Windows 10 After 12 Hours

### ISSUE THREAD

**Issue by KenACollins** [carbon-design-system/carbon-website#2115]
## Cannot Install carbon-website with Yarn on Windows 10

Hi, I have spent 12 hours so far on the `yarn install` step trying to install carbon-website on my Windows 10 machine and it has just been one roadblock after another. I resolve an issue only to be faced with another. Because there are so many package dependencies, it looks to me that things have changed since you last built the project on Windows 10, and I would like you to start fresh on a new Windows machine and see if you have any success.

### Yarn 2 Issues
For starters, you depend heavily on Yarn even though npm is the standard tool. Okay, fine but did you know that Yarn 2 is the current version and is nothing like Yarn 1? Workaround tips on Stack Overflow like how to turn off the SSL check for local certificates...

`yarn config set "strict-ssl" false -g`

...do not work with Yarn 2. I had to relent and install Yarn 1.22.5 with:

`npm install yarn@1.22.5 -g`

Another issue I ran into with Yarn is that it times out during installation, then blames the user with the following error:

`info There appears to be trouble with your network connection. Retrying…`

The fix is to change the `yarn install` command cited in your README.md file to be:

`yarn install --network-timeout 100000`

### Gatsby Issues
During the `yarn install` step, it failed for lack of Python 2.7 which had something to do with `node-gyp`. I don't like the idea of installing an old version of Python, but I had no choice, the error message I got cited Python 2 code:

`gyp ERR! stack Error: Command failed: C:\Python\Python39\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];`

An online search led me to issue this command which can only be issued from a Windows PowerShell prompt launched as Administrator:

`npm install --global windows-build-tools`

I also got errors about `gatsby-plugin-sharp` not being installed. There does not appear to be any clear way to resolve this.

### Missing Images
There are missing images in the code base that cause errors when pngquant is executed on them.

### pngquant Issues
The latest errors I am getting center on pngquant:

```
 !! unable to get local issuer certificate
 !! pngquant pre-build test failed
  i compiling from source
  × Error: pngquant failed to build, make sure that libpng-dev is installed
    at C:\Git Repositories\personal\carbon-website\node_modules\bin-build\node_modules\execa\index.js:231:11
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Promise.all (index 0)
```

The error message includes an issue with a local certificate and unfortunately the earlier yarn command to ignore this is not working here, even after I tried issuing the following:

`set NODE_TLS_REJECT_UNAUTHORIZED=0`

The error also suggests installing `libpng-dev`. Yarn does not maintain a catalog of packages, to my knowledge, but when I check npmjs.com and search on `libpng-dev` I learn there is no such package. How I am supposed to install it is a mystery.

I tried all of the following commands with no success:

```
yarn add pngquant -W
yarn add pngquant-bin -W
npm install imagemin-pngquant@5.0.1 --save
```

I added this to my resolutions section of package.json but no success:

```
"resolutions": {
    "image-webpack-loader/imagemin-pngquant": "5.0.1"
}
```

### Bottom Line
Every person's experience is bound to be different but the bottom line is that I strongly feel that no one at IBM has tried to install the carbon-website repo from scratch on a Windows 10 machine recently. A person like myself should not be facing so many roadblocks with no success after 12 hours of effort.

**UPDATE 2/2/21:** The README.md file does not state that this project can only be installed on a Mac, but I tried it on a  MacBook Pro laptop running macOS Catalina 10.15.7 and the whole process went smoothly with no errors and I was able to launch the website with the `yarn dev` command. This makes me highly suspicious that the IBM team supporting Carbon Design System and its carbon-website repo have not focused at all on the Windows 10 platform.
